test/specs/entityDefault.js

```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import expected from './../../data/expected.json';
+import {getRowCount, reset, userLogin} from "../../helper/methods";
+import testData from "../../data/testData.json";
+import selectors from "../../data/selectors.json";
+
+describe('DEFAULT', () => {
+    before(() => {
+        browser.maximizeWindow();
+        browser.url('/');
+        userLogin(testData.userLogin.loginName2, testData.userLogin.password2);
+        reset();
+    });
+
+    it('Should create default record with default values', () => {
+        $(selectors.default.sidebarDefault).click();
+        $(selectors.default.addNewRecordButton).click();
+        const saveButton =  $(selectors.default.saveButton);
+        saveButton.scrollIntoView();
+        saveButton.click();
+        expect($(selectors.default.firstRowStringCell)
+            .getText().includes(expected.default.defaultStringCellValue))
+            .to.equal(true);
+    });
+
+    it('Should create default record with new values', () => {
+        $(selectors.default.sidebarDefault).click();
+        $(selectors.default.addNewRecordButton).click();
+        $(selectors.default.inputString).click();
+        $(selectors.default.inputString).setValue(testData.default.string);
+        $(selectors.default.inputText).click();
+        $(selectors.default.inputText).setValue(testData.default.text);
+        $(selectors.default.inputInt).click();
+        $(selectors.default.inputInt).setValue(testData.default.integer);
+        $(selectors.default.inputDecimal).click();
+        $(selectors.default.inputDecimal).setValue(testData.default.decimal);
+        const saveButton =  $(selectors.default.saveButton);
+        saveButton.scrollIntoView();
+        saveButton.click();
+        expect($(selectors.default.secondRowStringCell)
+            .getText().includes(expected.default.defaultStringCellNewValue))
+            .to.equal(true);
+        expect(getRowCount()).to.equal(expected.common.twoRecords);
+    });
+
+    it('Should edit created record', () => {
+        $(selectors.default.secondRowActionMenuBtn).click();
+        browser.waitUntil(() => {
+            return $(selectors.default.secondRowActionMenuShow)
+                .getAttribute("class") === "dropdown pull-left show";
+        });
+        $(selectors.default.secondRowEditBtn).click();
```

Comment:
use method clickActionEdit(number) from methods.js
```suggestion
       clickActionEdit(2); 
```